docs/usage/configuration.rst

```diff
@@ -167,6 +167,22 @@ Default::
 
 .. setting:: FIREWALL_DIR
```

Comment:
And this needs to go after the `TEXTFSM_TEMPLATE_DIR` section, right above the heading for `FIREWALL_DIR`. Ya dig?